pkg/apis/camel/v1/integration_types_support.go

```diff
@@ -116,6 +116,11 @@ func (in *IntegrationSpec) AddDependency(dependency string) {
 	in.Dependencies = append(in.Dependencies, dependency)
 }
 
+// AddConfigurationProperty adds a new configuration property.
+func (in *IntegrationSpec) AddConfigurationProperty(confValue string) {
```

Comment:
`.integration.spec.configuration` is deprecated. We should not use it any longer.